votrfront/js/MojeSkuskyPage.js

```diff
@@ -94,11 +94,147 @@ export var MojeSkuskyPageContent = React.createClass({
     </table>;
   },
 
+  renderSkuskyExport() {
+    var cache = new CacheRequester();
+    var {zapisnyListKey} = this.props.query;
+
+    var vidim = cache.get('get_vidim_terminy_hodnotenia', zapisnyListKey);
+
+    if (!cache.loadedAll) {
+      return <Loading requests={cache.missing} />;
+    }
+
+    if (!vidim) {
+      return <p></p>; // intentionally left blank
+    }
+
+    var terminyPrihlasene = cache.get('get_prihlasene_terminy', zapisnyListKey);
+    var terminyVypisane = cache.get('get_vypisane_terminy', zapisnyListKey);
+
+    if (!terminyPrihlasene || !terminyVypisane) {
+      return <Loading requests={cache.missing} />;
+    }
+    
+
+    var terminy = [];
+    for (var i in terminyPrihlasene) {
+      // pick only future exams
+      var termin = terminyPrihlasene[i];
+      var now = new Date();
+      var now_year = now.getFullYear();
+      var now_month = now.getMonth() + 1; // getMonth returns range 0..11
+      var now_day = now.getDate(); // Date, not Day
+
+      var [tday, tmonth, tyear] = termin.datum.split(".");
+      tyear = parseInt(tyear);
+      tmonth = parseInt(tmonth);
+      tday = parseInt(tday);
+
+      if (tyear > now_year || (tyear == now_year && tmonth > now_month) || (tyear == now_year && tmonth == now_month && tday >= now_day)) {
```

Comment:
Hm :-/
Žeby radšej predsalen exportovať všetky termíny? Ušetril by sa tým tento hack, a nemalo by to vadiť AFAICT (staré skúšky tiež bude vidno v kalendári ale snáď to nebude nikoho trápiť).